scylla/src/utils/pretty.rs

```diff
@@ -51,7 +52,11 @@ where
             CqlValue::Inet(i) => write!(f, "'{}'", i)?,
             CqlValue::SmallInt(si) => write!(f, "{}", si)?,
             CqlValue::TinyInt(ti) => write!(f, "{}", ti)?,
-            CqlValue::Varint(vi) => write!(f, "{}", vi)?,
+            CqlValue::Varint(vi) => write!(
+                f,
+                "{}",
+                BigInt::from_signed_bytes_be(vi.to_signed_bytes_be_slice())
```

Comment:
I'd like to get rid of the dependency on num_bigint v3 here. I'd rather not reimplement printing bigint values in base 10 as the algorithm is not that simple (at least the one in num_bigint itself isn't), but maybe we can just print in hexadecimal here?

Preferably, this should be done in a CQL-compatible syntax (i.e. you can put the value formatted in this way into a query string as a literal and it should work), but it's not mandatory if it needs something complicated like printing in base 10.